services/elasticloadbalancing/src/main/resources/codegen-resources/customization.config

```diff
@@ -138,7 +137,7 @@
         "TooManyAccessPointsException" : "TooManyLoadBalancersException",
         "InvalidEndPointException" : "InvalidInstanceException"
     },
-    "sendEmptyAutoConstructedListAsEmptyList" : {
+    "sendNullAsEmptyListMembers" : {
```

Comment:
You know we may be able to kill this too. It's marked as required so customers are supposed to send an explicitly empty list to wipe out the load balancers. 

http://docs.aws.amazon.com/elasticloadbalancing/2012-06-01/APIReference/API_SetLoadBalancerPoliciesOfListener.html